doc/tutorials/12-constant_pH/12-constant_pH.ipynb

```diff
@@ -105,19 +105,23 @@
     "import espressomd\n",
     "from espressomd import reaction_ensemble\n",
     "\n",
-    "# class for working with SI units (this example works only with a limited set of units)\n",
-    "from si_units import SIunits\n",
-    "# Initialize the class by supplying the SI values of some internal units\n",
-    "# sigma=0.40 nm is a commonly used  particle size in coarse-grained simulations\n",
-    "SI=SIunits(sigma_in_nm=0.355, T_in_K=293.15);\n",
+    "\n",
+    "import scipy.constants # physical constants\n",
+    "import pint # class for working with units and dimensions\n",
```

Comment:
This is a desired change. However this breaks the tests of the tutorials. Please also update the sphinx documentation regarding the installation requirements.

@Kai we need dockerfiles with pint installed. How do we proceed here. Update the Docker images?

Reply:
`pint` was installed in docker and the requirements updated in 4d655b8

Reply:
I would drop the comment, or say "module for working with units and dimensions"

Reply:
ok